code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm

```diff
@@ -435,5 +435,5 @@
 	var/armor_block = run_armor_check(T, "melee")
 	var/armor_soak = get_armor_soak(T, "melee")
 	T.apply_damage(20, HALLOSS,, armor_block, armor_soak)
-	if(prob(33))
+	if(prob(99)) //for that 1% goodness
```

Comment:
99% is probably too much but a 1/10 chance of failure is probably okay. Make it 90.

Only 1/3 chance of success is dumb.